guides/common/attributes.adoc

```diff
@@ -84,7 +84,7 @@ ifeval::["{build}" == "satellite"]
 :package-install: yum install
```

Comment:
@spetrosi Hi Sergei, similar to package-remove, does package-install and package-update also need updating with satellite-maintain packages install/update? Everything else looks good to me.

Reply:
Good question. Technically it makes sense because you can run `satellite-maintain packages update`, but we do not use package updates in our docs. This is because the `satellite-maintain packages install` command does the same thing - checks if the package is installed and updates it to a latest version if it is not the latest version already. So we do not need attributes for an update.

Reply:
@spetrosi OK, understood - thanks.